bedrock/settings/base.py

```diff
@@ -453,10 +453,15 @@ def JINJA_CONFIG():
 # This can be changed to use session once we do add a database.
 MESSAGE_STORAGE = 'django.contrib.messages.storage.cookie.CookieStorage'
 
-EMAIL_BACKEND = config(
-    'EMAIL_BACKEND',
-    default='django.core.mail.backends.console.EmailBackend' if DEBUG else
+
+def lazy_email_backend():
```

Comment:
> If you don't make it lazy, you either can't override it from local.py (if you put it after the from .local import *) or you can't default it based on the DEBUG setting in local.py, unless I'm mistaken.

No, you're right. You'd have to do some nasty `if 'SOME_SETTING` not in globals():` trick, and using `lazy` seems less bad.

Also you're right about pulling settings back into a single file, well, plus the separate static_media file. My vote would be to keep that separate.